app/components/Common/Loader/Index.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import { View, Text, Modal, StyleSheet, ActivityIndicator } from 'react-native';
+import { Colors } from '../../../styles/colors';
 
 export default function Loader({ isLoaderShow }) {
   return (
     <Modal transparent visible={isLoaderShow}>
       <View style={styles.downloadModalContainer}>
         <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-          <ActivityIndicator size="large" color="#89B53A" />
+          <ActivityIndicator size="large" color={Colors.PRIMARY} />
```

Comment:
Got a crash when trying to sign in:
```
TypeError: undefined is not an object (evaluating '_colors.Colors.PRIMARY')

This error is located at:
    in Loader (at MainScreen/index.js:204)
```

Reply:
Solved in this commit https://github.com/Plant-for-the-Planet-org/treemapper/pull/207/commits/2b5a88d168dc9100541e1687bcbfd5f44c4ef121